runtime/shared_common/CompositeCache.cpp

```diff
@@ -1709,6 +1711,14 @@ SH_CompositeCacheImpl::startup(J9VMThread* currentThread, J9SharedClassPreinitCo
 	}
 	if (rc == CC_STARTUP_OK) {
 		_started = true;
+		if (NULL == cacheMemory) {
+			Trc_SHR_CC_startup_getCacheUniqueID_before(currentThread, getCreateTime(), getMetadataBytes(), getClassesBytes(), getLineNumberTableBytes(), getLocalVariableTableBytes());
+			const char* uniqueId = getCacheUniqueID(currentThread);
+			Trc_SHR_CC_startup_getCacheUniqueID_after(currentThread, strlen(uniqueId), uniqueId);
```

Comment:
If `uniqueId` is zero terminated, the tracepoint can just use `%s` and not take a length.